tests/functional/adapter/test_python_model.py

```diff
@@ -20,10 +20,22 @@ def project_config_update(self):
 
 models__simple_python_model = """
 import pandas
+import torch
+import spacy
 
 def model(dbt, spark):
     dbt.config(
         materialized='table',
+        submission_method='job_cluster',
+        job_cluster_config={
+            "spark_version": "7.3.x-scala2.12",
+            "node_type_id": "i3.xlarge",
+            "autoscale": {
+                "min_workers": 1,
+                "max_workers": 2
```

Comment:
The Databricks UI allows specifying just a single worker rather than multi-worker. Is that an option through the dbt config too?

Reply:
I think maybe just set it with max_workers to 1?  we just pass all of this to Databricks API so it is whatever they support

Reply:
Within the UI, it always goes up to max workers of 2 when I choose the "Multi node" option first. If I remember correctly, it would allow me to choose a max of 1, but then it would automatically bump it up to 2 later (I'll update this comment if I confirm or deny this later).

The only way to get a maximum of 1 for me was to choose the "Single node" option first:

<img width="222" alt="image" src="https://user-images.githubusercontent.com/44704949/193304235-50b506cb-a334-45da-a6ad-fd69ae716e93.png">

Reply:
https://docs.databricks.com/dev-tools/api/latest/clusters.html#examples I can update it to single node